### PR TITLE
fix: fix log loading with compatibility

### PR DESCRIPTION
**Pull request by minz1027**
Now UI is not loading logs with 1000 lines per file if it's more than 10000 lines. Because we don't know where to start to figure the lines per file if it's not requesting from line 0.

Adding an additional call to always fetch page 0 and set lines per file based on that.

Related to: https://github.com/screwdriver-cd/screwdriver/issues/1114

### CODE REVIEW

File: plugins/builds/steps/logs.js [hunk 1]
```diff
@@ -46,6 +46,33 @@ async function fetchLog({ baseUrl, linesFrom, authToken, page, sort }) {
     });
 }
 
+/**
+ * [getMaxLines description]
```
Comment (tkyi) [inline plugins/builds/steps/logs.js]: Missing description

File: plugins/builds/steps/logs.js [hunk 2]
```diff
@@ -46,6 +46,33 @@ async function fetchLog({ baseUrl, linesFrom, authToken, page, sort }) {
...
+ * @method getNumberPerFile
+ * @param  {[type]}         baseUrl           [description]
+ * @param  {[type]}         authToken         [description]
+ * @param  {String}         [sort='ascending' }]            [description]
```
Comment (tkyi) [inline plugins/builds/steps/logs.js]: extra curly brace

File: plugins/builds/steps/logs.js [hunk 3]
```diff
@@ -63,23 +90,21 @@ async function loadLines({
     authToken,
     pagesToLoad = 10,
     sort = 'ascending',
-    maxLines = MAX_LINES_SMALL
+    maxLines
```
[the remark this answers is not in the document]
Comment (minz1027) [reply plugins/builds/steps/logs.js]: fixed~

File: plugins/builds/steps/logs.js [hunk 4]
```diff
@@ -46,6 +46,33 @@ async function fetchLog({ baseUrl, linesFrom, authToken, page, sort }) {
     });
 }
 
+/**
+ * Returns number of lines per file based on lines returned for page 0
+ * @method getMaxLines
+ * @param  {Object}     config
+ * @param  {String}     config.baseUrl             URL to load from (without the .$PAGE)
+ * @param  {String}     config.authToken           Bearer Token to be passed to the Store
+ * @return {Promise}                          [description]
```
Comment (tkyi) [inline plugins/builds/steps/logs.js]: Still missing description here

**Pull request merged**